lib/auth_web/controllers/people_controller.ex

```diff
@@ -0,0 +1,45 @@
+defmodule AuthWeb.PeopleController do
+  @moduledoc """
+  Defines People controller functions
+  """
+  use AuthWeb, :controller
+
+  @doc """
+  `index/2` lists all the people who have authenticated with the auth app.
+  """
+  def index(conn, _params) do
+    # should be visible to superadmin and people with "admin" role
+    if conn.assigns.person.id == 1 do
+      render(conn, :index,
+        people: Auth.Person.list_people(),
+        roles: Auth.Role.list_roles(),
+        statuses: Auth.Status.list_statuses()
+      )
+
+      # Note: this can easily be refactored to save on DB queries. #HelpWanted
```

Comment:
Creating a `Records` or `Users` context that can fetch all three might be useful here

Reply:
Yeah, very happy to refactor code if it's too complex to understand/maintain. 👍 
That can be done as part of #104 

But please note: we aren't going to refer to **`people`** as "Users" _anywhere_ in the code
because we don't want people in the team/community referring to people as "Users".
See: https://github.com/dwyl/app/issues/33#issuecomment-689655311
Sorry for not making this clear sooner.